src/core/solvers/intro.cpp

```diff
@@ -245,7 +245,8 @@ double SolverIntro::calcDiff() {
         const std::shared_ptr<crocoddyl::ActionDataAbstract>& data = datas[t];
         if (model->get_nu() > 0 && model->get_nh() > 0) {
           Hu_lu_[t].compute(data->Hu);
-          YZ_[t] << Hu_lu_[t].matrixLU().transpose(), Hu_lu_[t].kernel();
+          YZ_[t] << (Hu_lu_[t].permutationP() * data->Hu).transpose(),
+              Hu_lu_[t].kernel();
```

Comment:
This implementation might allocate dynamic data in some compilers. Let's rewrite it as follows:
```suggestion
          YZ_[t].leftCols(Hu_rank_[t]).noalias() = (Hu_lu_[t].permutationP() * data->Hu).transpose();
          YZ_[t].rightCols(nu - Hu_rank_[t]) = Hu_lu_[t].kernel();
```